parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -1,3 +1,31 @@
 #!/bin/bash
 
-# Placeholder for future CIS-related shell utilities
\ No newline at end of file
+# Placeholder for future CIS-related shell utilities
+
+
+assignFilePermissions() {
```

Comment:
I don't see where `applyCIS` is called. Is this still for future usage?

Reply:
Same thing, #1013 does the calling. I marked this PR as WIP since it's not ready as-is.

Reply:
Done